Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/FilesStepDefinitions.cs

```diff
@@ -68,5 +86,105 @@ public async Task WhenISendsAGetRequestToTheFilesIdEndpoint(string identityName,
         _responseContext.WhenResponse = await identity.Files.GetFileMetadata(file.Id);
     }
 
+    [When($"{RegexFor.SINGLE_THING} sends a PATCH request to the /Files/{RegexFor.SINGLE_THING}.Id/ClaimFileOwnership with token {RegexFor.SINGLE_THING}.OwnershipToken")]
+    public async Task WhenISendsAPatchRequestToTheFilesFIdClaimFileOwnershipFOwnershipTokenEndpoint(string identityName, string fileName, string fileName2)
+    {
+        var identity = _clientPool.FirstForIdentityName(identityName);
+        var fileId = _filesContext.Files.FirstOrDefault(f => f.Key == fileName).Value?.Id ?? fileName;
+        var token = _filesContext.FileNameToOwnershipToken.FirstOrDefault(t => t.Key == fileName2).Value ?? "NonExistingTokenXXXX";
+        var request = new ClaimFileOwnershipRequest { FileOwnershipToken = token };
+        _responseContext.WhenResponse = _claimFileOwnershipResponse = await identity.Files.ClaimFileOwnership(fileId, request);
+        if (!_claimFileOwnershipResponse.IsError) _filesContext.FileNameToOwnershipToken[fileName] = _claimFileOwnershipResponse.Result!.NewOwnershipToken;
+        //reload the file into the context - to resemble the currently stored stage
```

Comment:
We usually put a space between "//" and the first word of a comment. And we start the comment with an uppercase letter. Same goes for the other comments you wrote
```suggestion
        // Reload the file into the context - to resemble the currently stored stage
```